### ISSUE TITLE
Stop iterating on the content that is 404'd or DMCA'd

### ISSUE THREAD

**Issue by viiibe** [timhutton/twitter-archive-parser#93]
Some content that is present in the archives either does not exist anymore due to it being deleted by the original uploader, or it is taken down via DMCA claims. The tool clearly omits the exceptions (as 'Download failed with status "404 Not Found"' and 'Download failed with status "403 Forbidden"' respectively), with the 403 one clearly specifying that the content in question has been struck by DMCA. Iterating through such content multiple times is a waste of time, and such media can be skipped to save time.

**Comment by chibiconsulting** [comment]
Agree. Was going to raise this issue myself. Thanks for the well written issue.
